### PR TITLE
MM-47449 : Avoid passing global state into actions/views/create_comment.tsx in mattermost webapp

### PR DESCRIPTION
**Pull request by pradeepmurugesan**
#### Summary
avoid multiple getState calls in `actions/views/create_comment.tsx`

#### Ticket Link
Fixes - https://github.com/mattermost/mattermost-server/issues/21348

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by m1lt0n — APPROVED** [review]
Looks good!

**Review by M-ZubairAhmed — COMMENTED** [review]
I do not see what has been changed here. This file doesnt seem to have call to `stores/redux_store` 🤔

File: actions/views/create_comment.tsx [hunk 1]
```diff
@@ -72,7 +73,7 @@ export function makeOnMoveHistoryIndex(rootId: string, direction: number) {
             dispatch(moveHistoryIndexForward(Posts.MESSAGE_TYPES.COMMENT as 'comment'));
         }
 
-        const nextMessageInHistory = getMessageInHistory(getState());
+        const nextMessageInHistory = getMessageInHistory(state);
```
Comment (koox00) [inline actions/views/create_comment.tsx:76]: I'm a bit skeptical on this one, are we sure that we don't want to pass in `getMessageInHistory` the updated `state`, do the above actions change what's returned?

**Review by koox00 — CHANGES_REQUESTED** [review]
Blocking merging the PR until we an answer the question about `getMessageInHistory`

**Comment by nevyangelova** [comment]
Sorry for the long wait on this, I updated the ticket and have been talking to @pradeepmurugesan in DM as well. We would ideally be calling getState as long as it's coming from a thunk inside the component. We don't want to import it from stores/redux_store

**Pull request closed without merging**